secant/Dependencies/LogsHandler/LogsHandlerLive.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  LogsHandlerLive.swift
+//  secant-testnet
+//
+//  Created by Lukáš Korba on 30.01.2023.
+//
+
+import Foundation
+import ComposableArchitecture
+
+extension LogsHandlerClient: DependencyKey {
+    static let liveValue = LogsHandlerClient(
+        exportAndStoreLogs: { tempSDKDir, tempTCADir, tempWalletDir in
+            async let sdkLogs = LogsHandlerClient.exportAndStoreLogsFor(key: LoggerConstants.sdkLogs, atURL: tempSDKDir)
```

Comment:
does it let you do this?

` async let sdkLogs = .exportAndStoreLogsFor(key: LoggerConstants.sdkLogs, atURL: tempSDKDir)`